src/utils/application.data.ts

```diff
@@ -125,6 +142,23 @@ export const findBoOrMo = (appData: ApplicationData, boMoType: string, id: strin
 export const checkGivenBoOrMoDetailsExist = (appData: ApplicationData, boMoType: string, id: string): boolean =>
   findBoOrMo(appData, boMoType, id) ? true : false;
 
+export const setBoNocDataAsArrays = (data: ApplicationDataType) => {
```

Comment:
Simply moved the function to a shared location in order to address Sonar duplicate code warnings. No code changes.